packages/@aws-cdk/aws-dynamodb/README.md

```diff
@@ -12,14 +12,14 @@ const table = new dynamodb.Table(this, 'Table', {
 
 ### Keys
 
-You can either specify `partitionKey` and/or `sortKey` when you initialize the
-table, or call `addPartitionKey` and `addSortKey` after initialization.
+`partitionKey` is required when a table is defined, `sortKey` is optional and can also be
```

Comment:
`sortKey` *cannot* be added using `addSortKey`.

Reply:
Fixed.